src/pages/home/ReportScreen.tsx

```diff
@@ -363,7 +363,7 @@ function ReportScreen({
         return reportIDFromRoute !== '' && !!report.reportID && !isTransitioning;
     }, [report, reportIDFromRoute]);
 
-    const isLoading = !ReportUtils.isValidReportIDFromPath(reportIDFromRoute) || !isSidebarLoaded || PersonalDetailsUtils.isPersonalDetailsEmpty();
+    const isLoading = !reportIDFromRoute || !isSidebarLoaded || PersonalDetailsUtils.isPersonalDetailsEmpty();
```

Comment:
@hungvu193 I also reverted the change in this PR https://github.com/Expensify/App/pull/38856 since the change below also solved that issue